.circleci/config.yml

```diff
@@ -747,6 +747,9 @@ workflows:
         equal: ["", << pipeline.parameters.gh_action_build_artifact_name >>]
     jobs:
       - e2e_testing:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Why is this needed?

Reply:
Because CircleCI by default will run on branches and not any tags. They treat SHAs the same way as tags. 
Refer to this thread https://discuss.circleci.com/t/v2-api-pass-git-sha-to-trigger-new-pipeline/39811